lambda/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+// Handler is the Lambda function handler
+func Handler(ctx context.Context, snsEvent events.SNSEvent) {
+	// stdout and stderr are sent to AWS CloudWatch Logs
+	log.Printf("Running Lambda function\n")
+
+	host := "192.168.2.7"
```

Comment:
This needs to be set to whatever the local machine IP address is, so instructions should be added to the README.